napari_arboretum/tree.py

```diff
@@ -9,32 +9,41 @@
 WHITE = np.array([255, 255, 255, 255], dtype=np.uint8)
 RED = np.array([255, 0, 0, 255], dtype=np.uint8)
 
+# RGBA in range [0, 255]
+ColorType = Tuple[float, float, float, float]
+
+
+@dataclass
+class Annotation:
+    x: float
+    y: float
+    text: str
```

Comment:
I know before it was `tstr` which isn't great, but could this `text` be made something more descriptive? Even `label` would be better than `text` IMO as I'd at least know what it was for

Reply:
👍 `label` is a much better name, will update